src/pages/Home.js

```diff
@@ -1,5 +1,26 @@
+import { useNavigate } from "react-router-dom";
+
 const Home = () => {
-  return <div>This is Home</div>;
+  let Navigate = useNavigate();
+  return (
+    <div className="homepage-container">
+      <div className="text">
+        <p>
+          <span className="heading-sm">Ready to play?</span>Let's get Started
+        </p>
+        <button onClick={() => Navigate("/categories")} className=" btn-cta">
+          Let's Go
+        </button>
+      </div>
+      <div className="img-container">
+        <img
+          className="img-responsive"
+          src="https://i.postimg.cc/vZj3v65z/quiz.png"
```

Comment:
You can create a js file for storing and getting these types of static data.

Reply:
Will do